### DISCUSSION TITLE
ActivityPub icon

### DISCUSSION THREAD

**Discussion by Ancurio** [ardacarofficial/links-website#15] [Suggestions and Opinions]
Hi, I made an activitypub icon to link to my firefish, based on this: https://en.m.wikipedia.org/wiki/File:ActivityPub-logo.svg

```xml
<!-- ActivityPub icon -->
<div class="header_svg_item">
  <a href="#link" target="_blank" rel="nofollow">
    <svg enable-background="new 0 0 24 24" viewBox="0 0 24 24">
      <path d="M 10.912419,4.45912 0.0284898,10.74265 v 2.51333 L 8.7354637,8.22933 v 10.05374 l 2.1769553,1.25685 z m -4.3534872,7.54038 -4.3539167,2.51332 4.3539167,2.5134 z m 6.5302282,-2.5137 4.353586,2.51351 -4.353586,2.51351 z m 0,-5.0269 10.883911,6.28367 v 2.51352 L 13.08916,19.5398 v -2.51346 l 8.707148,-5.02703 -8.707131,-5.02695 z"/>
      <path d="M 10.939454,4.4115 10.898434,4.4349 0,10.72592 v 2.57812 L 8.7070316,8.27869 v 10.01953 l 2.2324224,1.28906 z m 2.121093,0 v 0.0469 2.52929 l 8.681641,5.01172 -8.681641,5.01172 V 19.5873 L 24.000001,13.27285 V 10.72598 Z M 10.884766,4.5072 V 19.49157 L 8.7636722,18.26696 V 8.18102 L 0.056641,13.20641 v -2.44726 z m 2.232422,0 10.828125,6.25195 v 2.48047 l -10.828125,6.25195 v -2.44922 l 8.734375,-5.04297 -8.734375,-5.04297 z m -0.05664,4.92968 v 0.0488 5.07422 l 4.437501,-2.56055 z m 0.05664,0.0957 4.271485,2.46679 -4.271485,2.4668 z m -6.5312502,2.41796 -0.041016,0.0254 -4.3945312,2.5371 4.4355469,2.56055 z m -0.054687,0.0957 v 4.93164 L 2.2597665,14.51303 Z"/>
    </svg>
  </a>
</div>
```

![image](https://github.com/ardacarofficial/links-website/assets/1173822/4612acab-7bba-4110-87e9-1a95166cc5d2)

Then I noticed all your other logos are only using strokes with 1px (?) width, so tried to recreate that, but IMO it looks a bit awkward..

```xml
<!-- ActivityPub icon (no fill) -->
<div class="header_svg_item">
  <a href="#link" target="_blank" rel="nofollow">
    <svg enable-background="new 0 0 24 24" viewBox="0 0 24 24">
      <path d="M 11.433651,3.9115 10.712,4.3268249 0,10.511596 v 3.529322 L 8.3779119,9.2054834 V 18.323839 L 11.433651,20.0885 Z m 1.133217,0 v 0.8325292 2.696793 L 20.46368,12 12.566868,16.559178 V 20.0885 L 24.000519,13.486525 V 10.511596 Z M 10.47145,5.5784376 V 18.421562 L 9.3401127,17.767567 V 7.5385457 L 0.96220079,12.37398 v -1.306112 z m 3.057619,0 9.509249,5.4894304 v 1.862385 l -9.509249,5.491309 V 17.113571 L 22.388081,12 13.529069,6.8845499 Z m -0.962201,3.168497 V 9.581343 15.251186 L 18.199125,12 Z M 13.529069,10.413873 16.274723,12 13.529069,13.584248 Z m -6.284374,0.751719 -0.7216506,0.417204 -4.9106065,2.835861 5.6322571,3.251186 z m -0.9622008,1.666937 v 3.170376 L 3.5368394,14.416778 Z"/>
    </svg>
  </a>
</div>
```

![image](https://github.com/ardacarofficial/links-website/assets/1173822/eee14179-ed7d-42e4-81ae-80cb646652bf)

**Comment by ardacarofficial** [comment]
![image](https://github.com/ardacarofficial/links-website/assets/32874384/2b335707-17fe-46d6-91fb-bac75e05c855)

I tested the SVG code now. There is no problem in the code, that is, no code intervention breaks the SVG image, the SVG image looks weird because it is that way.

I will try to find a more polite SVG similar to this SVG.

**Comment by ardacarofficial** [comment]
@Ancurio Unfortunately, for some reason I didn't get the desired result. I tried redrawing the SVG in Figma, but again there is no difference in appearance. 

I am not locking the thread in case someone solves the problem or you solve the problem. If you solve the problem, please let me know under this thread.

**Reply by Ancurio** [reply]
What is your desired result? I'm a bit confused, because the 2nd sample I provided look exactly how the AP logo would look if you only rendered the outlines, at 1px thickness

**Reply by ardacarofficial** [reply]
You said it looked weird when you opened the topic, I don't know if it's because of the SVG, but it looks weird to me too.

Exactly what I mean is that it doesn't look good, as you said, I think the problem is SVG.
